tools/dm/shard-merge-scenario.md

```diff
@@ -0,0 +1,273 @@
+---
+title: DM 分片合并场景
+summary: 学习分片合并场景下如何使用 DM 同步数据
+category: tools
+---
+
+# Data Migration 分片合并场景
+
+本文介绍如何在分片合并场景中使用 Data Migration (DM)。使用场景中，三个上游 MySQL 实例的分库和分表数据需要同步至下游 TiDB 集群。
+
+## 上游实例
+
+假设上游库结构如下：
+
+- 实例 1
+
+    | 库 | 表 |
+    |:------|:------|
+    | user  | information, log_north, log_bak |
+    | store_01 | sale_01, sale_02 |
+    | store_02 | sale_01, sale_02 |
+
+- 实例 2
+
+    | 库 | 表 |
+    |:------|:------|
+    | user  | information, log_east, log_bak |
+    | store_01 | sale_01, sale_02 |
+    | store_02 | sale_01, sale_02 |
+
+- 实例 3
+
+    | 库 | 表 |
+    |:------|:------|
+    | user  | information, log_south, log_bak |
+    | store_01 | sale_01, sale_02 |
+    | store_02 | sale_01, sale_02 |
+
+## 同步需求
+
+1. 合并三个实例中的 `user`.`information` 表至下游 TiDB 中的 `user`.`information` 表。
+2. 合并三个实例中的 `user`.`log_{north|south|east}` 表至下游TiDB中的 `user`.`log_{north|south|east}` 表。
```

Comment:
```suggestion
2. 合并三个实例中的 `user`.`log_{north|south|east}` 表至下游 TiDB 中的 `user`.`log_{north|south|east}` 表。
```

Reply:
Also check all similar expressions and make sure to leave a space between English and Chinese characters.